models/issues/dependency_test.go

```diff
@@ -49,7 +49,7 @@ func TestCreateIssueDependency(t *testing.T) {
 	assert.False(t, left)
 
 	// Close #2 and check again
-	_, err = issues_model.ChangeIssueStatus(db.DefaultContext, issue2, user1, true)
+	_, err = issues_model.CloseIssue(db.DefaultContext, issue2, user1)
```

Comment:
Looks like we only test `issues_model.CloseIssue`. Do we need any tests on `issues_model.ReopenIssue` ?

Reply:
a869ae7088